docs/transport/braginskii.rst

```diff
@@ -0,0 +1,60 @@
+.. _braginskii:
+
+Braginskii
+**********
+
+The `~plasmapy.physics.transport.braginskii` subpackage includes a 
+class interface 
+`~plasmapy.physics.transport.braginskii.ClassicalTransport` with 
+multiple methods which are involved in modeling classical transport.
+
+.. topic:: Examples:
+
+   * :ref:`sphx_glr_auto_examples_plot_braginskii.py`
+
+.. _Classical transport models:
+
+Classical transport models
+==========================
+
+Broad overview of classical transport models implemented within 
```

Comment:
Having done some tinkering with Sphinx and observing how they do it at Sunpy and Astropy, I think the best thing to do to avoid duplication is to actually keep this in docstrings. So what I'll do is move a lot of our ClassicalTransport docstring to the braginskii.py module docstring (with your edits), refer to the module docstring from the class docstring, and... yeah. I think that's it. We'll see how it ends up working.

Reply:
> and condition 1 below is just a new condition for validity.

Should it be added to the first list, then? Maybe like this?
```rst

Keep in mind the following assumptions under which the transport equations
are derived:

1. The plasma is fully ionized, only consisting of ions and electrons.
   Neutral atoms are neglected.
2. Turbulent transport does not dominate
3. The velocity distribution is close to Maxwellian. This is guaranteed if:
    i. Collisional mean free path << gradient scale length along field
    ii. Gyroradius << gradient scale length perpendicular to field
4. The plasma is highly collisional: collisional frequency >> gyrofrequency
```

Reply:
Looks great!  I don't know if we can say guaranteed in 3, I think I found those orderings in the Stacey fusion plasmas textbook, I'm not sure how all-encompassing they are. Maybe we can say "This implies" rather than "this is guaranteed if".

Reply:
Sure! Let's do that.